praw/models/reddit/subreddit.py

```diff
@@ -67,16 +67,24 @@ class Subreddit(RedditBase, MessageableMixin, SubredditListingMixin):
     ======================= ===================================================
     Attribute               Description
     ======================= ===================================================
+    ``can_assign_link_flair`` Whether users can assign their own link flair.
```

Comment:
This likely is also causing some issues.

Reply:
Oh, OK. I'll fix that. Thanks.

Reply:
OK, e2794b1 should fix the Sphinx build, not sure about Pylint

Reply:
Hmm, some lines are too long. I'll shorten them.

Reply:
Cool, Travis passes!